docs/js/modal.md

```diff
@@ -49,11 +46,12 @@ Based on this markup, the component will get all content inside of `[data-modal]
 
 ## Options
 
-Modal provides three customizable options: `container`, `triggerClose` and `triggerOpen`. By default `.modal` is appended in the `body`, `triggerClose` and  `triggerOpen` are `null`.
+Modal provides some customizable options such as: `container`, `size`, `triggerClose` and `triggerOpen`. The default container option for the `.modal` class is to be appended in the `body`, the `size` is 'medium', `triggerClose` and  `triggerOpen` are `null`.
```

Comment:
vírgula antes do `and`
Não ficaria melhor usar `behaviour` ao invés de `option`? :thinking: